lsp-notifications.el

```diff
@@ -24,6 +24,11 @@
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-report-if-no-buffer t
```

Comment:
Shouldn't the default value be `nil`, since that's what the current behaviour is?

Reply:
The original behaviour was to preserve the errors but it was changed in https://github.com/emacs-lsp/lsp-mode/commit/d17c55448c26c8475a5e38354ca42d7875b239b4 and this CL is actually redoing this commit. IMO we should keep it as it is since https://github.com/emacs-lsp/lsp-mode/commit/d17c55448c26c8475a5e38354ca42d7875b239b4 was introduced as a potential workaround of https://github.com/emacs-lsp/lsp-mode/issues/390 and it should not be the default behaviour. See also https://github.com/emacs-lsp/lsp-mode/issues/390#issuecomment-416924475

Reply:
Ah, I didn't realize this was breaking `lsp-ui`. I'll merge this, then.